ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -162,16 +162,16 @@ func createCluster(context *cli.Context, rdwr config.ReadWriter, ecsClient ecscl
 
 	// Check if cluster is specified
 	if ecsParams.Cluster == "" {
-		return fmt.Errorf("Please configure a cluster using the configure command.")
+		return fmt.Errorf("Please configure a cluster using the configure command or with '--%s' flag", command.ClusterFlag)
```

Comment:
I removed the period because I was trying to follow the best practices. "Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation"
https://github.com/golang/go/wiki/CodeReviewComments#error-strings